packages/ember-htmlbars/lib/component.js

```diff
@@ -138,6 +139,21 @@ const Component = View.extend(TargetActionSupport, {
     }
   }),
 
+  templateForName(name, type) {
```

Comment:
The issue is I want to get these private APIs cleaned up early, I'm hoping no one complains and tries to argue templateForName is intimate but it is best I think to give people some time with canary.